utils/utils.go

```diff
@@ -565,6 +565,14 @@ func CustomizeEnvSSO(pts *corev1.PodTemplateSpec, instance *wlv1.WebSphereLibert
 	return nil
 }
 
+// Adds Service annotations to enable zone aware routing
+func CustomizeServiceAnnotations(svc *corev1.Service) {
```

Comment:
It'll be good to consider adding this util method to RCO utils and call it as part of CustomizeService (prior to annotation from CR are added, so customers can override if they wanted to). Semeru can call the same util method.

Reply:
Thanks, that will be useful for code sharing. I've moved to RCO and updated the dependency on this PR.